table/builder.go

```diff
@@ -133,10 +127,9 @@ func (b *Builder) addHelper(key []byte, v y.ValueStruct) {
 	if len(key) > 0 {
 		keyNoTs := y.ParseKey(key)
 		keyHash := farm.Fingerprint64(keyNoTs)
-		b.bloomFilter.Add(keyHash)
-		if b.enableHashIndex {
-			b.hashIndexBuilder.addKey(keyHash, uint32(len(b.baseKeysEndOffs)), uint8(b.counter))
-		}
+		// It is impossible that a single table contains 16 million keys.
```

Comment:
Skip different versions of the same key?

Reply:
I prefer to keep it simple if it does not make much difference.

Reply:
👌